NEWS

```diff
@@ -3,8 +3,16 @@ NEWS
 ====
 
 :Authors: Toshio Kuratomi, Luke Macken, Ricky Elrod, Patrick Uiterwijk, Ralph Bean
-:Date: 5 May 2017
-:Version: 0.10.x
+:Date: TBD
+:Version: 0.11.x
```

Comment:
I'd argue for a 1.0.x release here since it's quite non-backward compatible as change :D

Reply:
How about ½.11.0? :D

Reply:
unicode++